sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.3.1 (Unreleased)
+
+### Bug Fixes
+
+- Fixed the logic of `PhoneNumberIdentifier` to always maintain the original phone number string whether it included the leading + sign or not.
```

Comment:
As  `_shared` folder is copied across modalities, this change should be reflected in the changelog of other modalities (or at least those modalities that use `PhoneNumberIdentifier`)

Reply:
We should only mention for modalities that expose this functionality to public. Job router does not expose this to public, so we don't have to mention it in their changelog. In other modalities we didn't change anything in the shared folder, so no need in mentioning it.